frontend/base/src/main/java/com/virtuslab/gitmachete/frontend/resourcebundles/GitMacheteBundle.java

```diff
@@ -26,12 +29,17 @@ private GitMacheteBundle() {}
    *             but each parameter must be a non-null {@link String} and not just a nullable {@link Object}
    * @return the formatted string
    */
-  public static String format(@I18nFormatFor("#2") String format, String @MinLen(1)... args) {
+  public static @PolyTainted String format(@PolyTainted @I18nFormatFor("#2") String format, String @MinLen(1)... args) {
```

Comment:
`PolyTainted` in the sense that if param is `@Untainted` (surely non-HTML, in our usage), then the result will also be `@Untainted`; and vice versa, if param is `@Tainted` (maybe containing HTML, in our usage), then the result will also be `@Tainted`